src/results/resultUtils.ts

```diff
@@ -212,7 +221,130 @@ export function getTypeDef(type: string): RelTypeDef {
 
   return {
     type: 'Unknown',
-    name: type,
+  };
+}
+
+export function getTypeDefFromProtobuf(type: RelType): RelTypeDef {
+  if (
+    type.tag === Kind.CONSTANT_TYPE &&
+    type.constantType?.value &&
+    type.constantType?.relType
+  ) {
+    const typeDef = getTypeDefFromProtobuf(type.constantType.relType);
+
+    if (typeDef.type !== 'ValueType') {
+      const values = type.constantType.value.arguments.map(mapPrimitiveValue);
+      const value = convertValue(
+        typeDef,
+        values.length === 1 ? values[0] : values,
+      );
+
+      return {
+        type: 'Constant',
+        value: {
+          ...typeDef,
+          value,
+        } as RelTypedValue,
+      };
+    } else {
+      const value = unflattenContantValue(
```

Comment:
I think this is a typo `unflattenConstantValue` ?

Reply:
Hehe, I stared for one minute and couldn't see it :D Thx!